Assets/Car/Scripts/CarSoundScript.cs

```diff
@@ -0,0 +1,36 @@
+﻿using UnityEngine;
+
+public class CarSoundScript : MonoBehaviour
+{
+    private AudioSource _audioSource;
+    public AudioClip CollisionClip;
+    public AudioClip CheckpointClip;
+    public float maxEnginePitch = 2;
+    public float minEnginePitch = 1;
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        _audioSource = GetComponent<AudioSource>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKey(KeyCode.W))
```

Comment:
Misschien leuker om `Input.GetAxis("Vertical");` te gebruiken, om ook controller support te behouden

Reply:
Mischien kunnen we ook wel de snelheid van het voertuig opvragen inplaats van een trigger op w en s? 
Ik denk dat je daarmee ook voorkomt dat het geluid bijvoorbeeld blijft afspelen bij een collision.

Reply:
CarScript heeft een RPM variable, dus dit is zeker mogelijk :)